sky/engine/core/script/dart_controller.cc

```diff
@@ -106,6 +106,15 @@ void DartController::RunFromSnapshot(
       base::Bind(&DartController::DidLoadSnapshot, weak_factory_.GetWeakPtr()));
 }
 
+void DartController::RunFromSnapshotBuffer(const uint8_t* buffer, size_t size) {
```

Comment:
I think I could change it do so, but it didn't seem necessary. Also, RunFromSnapshot assumes Dart_CurrentIsolate() is null, which is not the case for my code. Not sure how much that matters.